### ISSUE TITLE
Load library from different path fails (LoadLibrary call fail).

### ISSUE THREAD

**Issue by nathan130200** [nefarius/Injector#31]
Basically when attempt to inject any library from different directory than injector.exe root, the calls to LoadLibraryW always fail. I looked in the code, seems the injector load only for relative paths to injector.exe.

Same as #26

**Comment by nefarius** [comment]
What's your example to reproduce?

**Comment by nathan130200** [comment]
The command `injector.exe -p <pid of game after successful start> -i "C:\Projects\Game Helpers\bin\Debug\game-helpers.dll"` and `injector.exe` in different path from `C:\Projects\Game Helpers\bin\Debug`, eg: `E:\Programs\utils\injector.exe`

Output same LoadLibraryW call failed in remote process.

If i place `game-helpers.dll` in same folder of injector.exe and call it directly `-i game-helpers.dll` works fine.

![image](https://github.com/nefarius/Injector/assets/14365254/88f32df5-e190-4ea4-be52-e9688c341ec7)

The example above is called after my launcher start game .exe file then get PID and inject that library. Since i need hook stuff in c++ functions before first call to `CreateMutexA` i must inject instantly into the game.

**Comment by nefarius** [comment]
Yeah that's definitely supposed to work like that, will investigate.

**Comment by ManoFx0000** [comment]
Hi, I am getting "Call to LoadLibraryW in remote process failed."

Tried using both 32bit and 64bit injectors still same issue 

Both DLL and Injector are in same folder still same issue .. 

I tried to put them in different drive But same folder still same issue .. 

Please help how can I solve this
